src/main/kotlin/com/autonomousapps/model/declaration/Configurations.kt

```diff
@@ -79,13 +79,16 @@ internal object Configurations {
   }
 
   @OptIn(ExperimentalStdlibApi::class)
-  private fun findVariant(variantSlug: String, supportedSourceSets: Set<String>): Variant? {
+  private fun findVariant(variantSlug: String, supportedSourceSets: Set<String>, hasCustomSourceSets: Boolean): Variant? {
     if (variantSlug.isNotEmpty() && !supportedSourceSets.contains(variantSlug)) return null
-
-    return if (variantSlug == Variant.TEST_NAME) {
+    return if (variantSlug.isEmpty()) {
```

Comment:
why `variantSlug.isEmpty()`?

Reply:
The empty string represents the `main` variant (I added a comment now). Or is that not always the case? It is for standard Java projects (because everything with a "variantSlug" is either "test" or "custom").

I moved this check for the "main" variant first in this list of checks, because in the "custom source set case" it will default to that further down.

There is still the "else" in the end that also falls back to `main`, but I think that's a case we should never reach (unless this is something in Android I am not aware of).